core/exceptions/ExceptionStackTraceDisplay.php

```diff
@@ -68,73 +68,75 @@ protected function displayException(Exception $exception)
             ? $this->developerError($exception, $msg, $code, $trace_details)
             : $this->genericError($msg, $code);
         // start gathering output
-        $output = '
-<div id="ee-error-message" class="error">
-    ' . $error_message . '
-</div>';
-        $styles = $this->exceptionStyles();
+        $output = '<div id="ee-error-message" class="error">
+            ' . $error_message . '
+        </div>';
```

Comment:
can you keep these line breaks as part of the formatting plz?

the only reason they are there and why i would like to keep them is because it makes it easier to find this content in the DOM if you are looking for it. The way you have it, `<div id="ee-error-message" class="error">` will get tacked onto the end of whatever other line of HTML is there like this:

```html
<div id="post-body-content">
	<div class="ee-admin-container">
		<div class="padding">
			<form action="https://makeeventsnotwar.com/wp-admin/admin.php?page=espresso_events&amp;route=default" id="Extend_Events_Admin_List_Table-table-frm" method="get"><div id="ee-error-message" class="error">
            Unable to find markup
        </div>
    <p class="search-box">
	<label class="screen-reader-text" for="toplevel_page_espresso_events-search-input">Search Events:</label>
	<input type="search" id="toplevel_page_espresso_events-search-input" name="s" value="">
		<input type="submit" id="search-submit" class="button" value="Search Events"></p>
		        <ul class="subsubsub">
```

which makes it hard to find if you are looking for `id="ee-error-message"`

Another solution would be to remove that markup from this file and place it in a template file and then load that as needed.

Reply:
🙇🏻‍♂️